front-end/src/App.tsx

```diff
@@ -96,6 +100,18 @@ const App = () => {
 													<Route path="/onchain" >
 														<OnChain/>
 													</Route>
+													<Route path="/referenda" >
+														<Referenda/>
+													</Route>
+													<Route path="/proposals" >
+														<Proposals/>
+													</Route>
+													<Route path="/motions" >
+														<Motions/>
+													</Route>
+													<Route path="/treasury" >
```

Comment:
This will clash with the above `treasury/:id`. Because `treasuries` would be awful, I suggest `treasury-proposals`, not awesome either, but less bad I think.